features/visitor_can_view_listing.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Visitor can view listing
+  As a Visitor
+  In order to find a suitable host for my cat
+  I want to be able to view listings on the landing page
+
+  Background:
+    Given the following listings exist
+    | name | location   | description | email        | phone_number | availability |
+    | Zane | Gothenburg | I'm nice    | zane@cat.com | 0701234567   | 1-10 may     |
+
+  Scenario:
+    When I visit the landing page
+    Then I should see "Zane"
+    And I should see "Gothenburg"
```

Comment:
You need to test that you see "Gothenburg" for "Zane"

Reply:
will do

Reply:
Well, I would try to formulate the test in a way that calls for use of `within` in order to make sure that not only is the text visible, but it's visible on the right listing.

Reply:
I guess that is what you asked for as well @oliverochman ?

Reply:
@tochman yes :+1: